dbms/src/Databases/DatabasesCommon.h

```diff
@@ -76,13 +76,13 @@ class DatabaseWithOwnTablesBase : public IDatabase
 public:
     bool isTableExist(
         const Context & context,
-        const String & table_name) const override;
+        const String & table_name) override;
```

Comment:
I don't like removing `const` from methods... maybe it is better to make cache `mutable`.